packages/ckeditor5-dev-env/lib/release-tools/utils/transform-commit/transform-commit-utils.js

```diff
@@ -39,31 +39,100 @@ const transformCommitUtils = {
+	linkToGithubUser( comment ) {
+		return comment.replace( /@([0-9A-Z_-]+\/?)/ig, ( matchText, nickName ) => {
+			if ( nickName.endsWith( '/' ) ) {
+				return matchText;
+			}
+
+			return `[@${ nickName }](https://github.com/${ nickName })`;
+		} );
 	},
 
 	/**
-	 * Changes references to issue to links that lead to the GitHub issue page.
+	 * Replaces reference to issue (#ID) with a link to the issue.
 	 *
-	 * @param {String} sentence
+	 * @param {String} comment
 	 * @returns {String}
 	 */
-	linkGithubIssues( sentence ) {
+	linkToGithubIssue( comment ) {
 		const packageJson = getPackageJson();
 		const issuesUrl = ( typeof packageJson.bugs === 'object' ) ? packageJson.bugs.url : packageJson.bugs;
 
 		if ( !issuesUrl ) {
 			throw new Error( `The package.json for "${ packageJson.name }" must contain the "bugs" property.` );
 		}
 
-		return sentence.replace( /#([0-9]+)/g, ( _, issueId ) => {
-			return `[#${ issueId }](${ issuesUrl }/${ issueId })`;
+		return comment.replace( /(.?)#([0-9]+)/g, ( matchText, charBeforeIssue, issueId ) => {
+			// Don't replace anything if the '#ID' belongs to another part of the comment.
+			if ( charBeforeIssue && /[A-Z0-9_-]/i.test( charBeforeIssue ) ) {
+				return matchText;
+			}
+
+			return `${ charBeforeIssue }[#${ issueId }](${ issuesUrl }/${ issueId })`;
+		} );
+	},
+
+	/**
+	 * Replaces reference to repository (organization/repository) with a link to the repository.
+	 *
+	 * If the reference contains an issue, link will lead to the issue.
+	 *
+	 * @param {String} comment
+	 * @returns {String}
+	 */
+	linkToGithubRepository( comment ) {
+		return comment.replace( /(@|\.)?([A-Z0-9-_/]+)(#(\d+)?)?/gi, ( matchText, charBeforeRepo, repository, issueMark, issueId ) => {
+			if ( !isValidRepository() ) {
+				return matchText;
+			}
+
+			if ( issueId ) {
+				return `[${ repository }#${ issueId }](https://github.com/${ repository }/issues/${ issueId })`;
+			}
+
+			return `[${ repository }](https://github.com/${ repository })`;
+
+			function isValidRepository() {
+				// If the repository starts with '@', it means the package link should lead to NPM.
+				if ( matchText.startsWith( '@' ) ) {
+					return false;
+				}
+
+				// If the repository starts with '.', it means the repository is part of other link.
+				if ( matchText.startsWith( '.' ) ) {
+					return false;
+				}
+
+				// If the issue hash (#) occurs in repository but the issue id misses, don't modify it.
+				if ( issueMark && !issueId ) {
+					return false;
+				}
+
+				// If the repository contains more than two slashes, it means the repository is a path.
+				return repository.split( '/' ).length === 2;
+			}
+		} );
+	},
+
+	/**
+	 * Replaces scoped package name with a link which will land to the package in NPM repository.
+	 *
+	 * @param {String} comment
+	 * @returns {String}
+	 */
+	linkToNpmScopedPackage( comment ) {
+		return comment.replace( /(@[A-Z0-9-_/]+)#?/gi, ( matchText, repository ) => {
```

Comment:
Again `(?!#)`

Reply:
And if you want to have exactly two segments separated with `/` just write this in the pattern.

Reply:
This function has been removed.